bulb.py

```diff
@@ -0,0 +1,59 @@
+
+'''Class that contains the set of rules around sensors'''
+class Bulb(object):
+
+    def __init__(self,  name, lightSensor, presenceSensors, homeSensor, wastage_function):
+        self.name = name
+        self.light_sensor = lightSensor
+        self.presence_sensors = presenceSensors
+        self.home_sensor = homeSensor
+        self.wastage_function = wastage_function
+
+
+    def isWasted(self, *args):
+        return self.wastage_function(self,*args)
+
+
+# Actual function implementation for the given lights
+
+def kitchenLampWasted(self):
+    # assumes self.presence_sensors = [desk_sensor, bed_sensor]
+    if not self.light_sensor.isOn():
+        return False, 'OFF'
+    elif not self.home_sensor.isHome():
+        return True, 'NOT_HOME'
+    elif self.presence_sensors[0].isAround():
```

Comment:
`presence_sensors[0]` is literally what I wrote on the paper to illustrate the point, but in the implementation of these methods, instead of saying in the comment 
```python
# assumes self.presence_sensors = [desk_sensor, bed_sensor]
```
you could actually just rename these to improve the readability at the start of the method:
```python
desk_sensor = self.presence_sensors[0]
bed_sensor = self.presence_sensors[1]
```